.jshintignore

```diff
@@ -80,6 +80,8 @@ tv_apps/smart-home/bower_components/**
 tv_apps/smart-settings/bower_components/**
 tv_apps/smart-system/bower_components/**
 tv_apps/notification/bower_components/**
+tv_apps/fling-tutorial/bower_components/**
+tv_apps/fling-tutorial/offline_content/bower_components/**
```

Comment:
1.`bower_components` theoretically should  be installed from bower.json, and you didn't included them. If you don't quite know about bower components I would suggest making a brief study.

2.I don't think making offline_content also a bower component is a good idea. Please don't just put everything under `offline_content`.